### PR TITLE
add step and correct temporal extent

### PR DESCRIPTION
**Pull request by dthiex**
This PR updates the temporal extend to represent the currently available data more better. Once newer data is available this can be updated. 

Info about `step` in datacube extension here: https://github.com/stac-extensions/datacube#temporal-dimension-object

### CODE REVIEW

File: collections/vegetation-indices.yaml [hunk 1]
```diff
@@ -127,7 +127,7 @@ Extent:
     interval:
       -
         - '2016-10-01T00:00:00Z'
-        - null
+        - '2021-02-28T00:00:00Z'
```
Comment (Dorothyrono) [inline collections/vegetation-indices.yaml:130]: Adjust the hr,min,sec part  , `'2021-02-28T00:00:00Z' ` leaves out some data

 something like  '2021-02-28T23:60:60Z'  would capture all data if i am not wrong

**Comment by Dorothyrono** [comment]
Seasonal Trajectories is left out 
step should be 10 days

File: collections/vegetation-indices.yaml (hunk 1 above)
Comment (dthiex) [reply collections/vegetation-indices.yaml:130]: Changed to `2021-03-01T00:00:00Z` as I feel it's more correct.

**Comment by dthiex** [comment]
> 
> 
> Seasonal Trajectories is left out step should be 10 days

I didn't add this on purpose as it's not always 10 days (e.g. you get one on February 21 and then the next already on March 1st). If this is added the spacing between observations at the same location should always be the same.

**Comment by Dorothyrono** [comment]
> > Seasonal Trajectories is left out step should be 10 days
> 
> I didn't add this on purpose as it's not always 10 days (e.g. you get one on February 21 and then the next already on March 1st). If this is added the spacing between observations at the same location should always be the same.

Ok makes sense now

File: collections/vegetation-indices.yaml (hunk 1 above)
Comment (Dorothyrono) [reply collections/vegetation-indices.yaml]: alright .. looks good

lets merge

**Review by Dorothyrono — APPROVED** [review]

**Pull request merged**